client/app/visualizations/treemap/treemap.less

```diff
@@ -0,0 +1,48 @@
+.node {
```

Comment:
Can you change these classes to be more specific/scoped? `.node`, `rect`, `text` and some others are too generic and may affect other parts of the app.

Reply:
My bad. Done.